core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -313,42 +314,59 @@ object TopicCommand extends Logging {
     }
 
     override def describeTopic(opts: TopicCommandOptions): Unit = {
-      val topics = getTopics(opts.topic, opts.excludeInternalTopics)
-      ensureTopicExists(topics, opts.topic, !opts.ifExists)
+      // if topicId is provided and not zero, will use topicId regardless of topic name
+      val useTopicId = opts.topicId.map(Uuid.fromString).nonEmpty
```

Comment:
Based on the above comment, does not it need to handle `UUID.ZERO_UUID` too?

Reply:
good point!

Reply:
Thank you, I was careless about this case, I add the following fix:
1. When --topic and --topic-id are both specified, I will print a warning message that saying that "topicId will be used if both are specified and topicId is not zero"
2. Filter ZERO UUID before handling in TopicCommand.